tuhi/export.py

```diff
@@ -82,27 +82,44 @@ class JsonSvg(ImageExportBase):
     _base_pen_width = 0.4
     _pen_pressure_width_factor = 0.2
 
+    # Change this value down to reduce size, change it up to improve accuracy
+    width_precision = 10
```

Comment:
what's the unit of this? We should have that for `base_pen_width` too come to think of it, I think that one is in mm

Reply:
It's in the unit of width, which is pixels. So this is a precision of a tenth of a pixel and might be proportional to the base pen width, I couldn't be sure.